### PR TITLE
Session pagination e2e tests

### PR DESCRIPTION
**Pull request by mikemountain**
Attempting to cancel sessions is flaky so these tests avoid the cancellation part of the process

### CODE REVIEW

File: testing/internal/e2e/tests/base/paginate_session_test.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	e2e.MaybeSkipTest(t)
+	c, err := loadTestConfig()
+	require.NoError(t, err)
+
+	client, err := boundary.NewApiClient()
+	require.NoError(t, err)
+	ctx := context.Background()
+	sClient := sessions.NewClient(client)
+	newOrgId := boundary.CreateNewOrgApi(t, ctx, client)
+
```
[the remark this answers is not in the document]
Comment (johanbrandhorst) [reply testing/internal/e2e/tests/base/paginate_session_test.go:107]: I guess this could be why the tests are failing?

**Review by johanbrandhorst — APPROVED** [review]
LGTM when the cleanup is added

File: testing/internal/e2e/tests/base/paginate_session_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (mikemountain) [reply testing/internal/e2e/tests/base/paginate_session_test.go:107]: Some CLI/API things as well, currently fixing

**Review by moduli — APPROVED** [review]
I'm still curious why the cancel doesn't work, but I'll investigate that on the side.

**Pull request merged**